.ci/env/derecho.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+echo "Setting up derecho environment"
+workingDirectory=$PWD
+. /etc/profile.d/z00_modules.sh
+echo "Loading modules : $*"
+cmd="module purge"
+echo $cmd && eval "${cmd}"
+
+# We should be handed in the modules to load
+while [ $# -gt 0 ]; do 
+  cmd="module load $1"
+  echo $cmd && eval "${cmd}"
+  shift
+done
+
+#  Go back for asinine reasons of HPC config changing your directory on you
```

Comment:
Perhaps there's a more diplomatic way of stating this. Regardless, is it always the case that the working directory is changed (in which case, we can unconditionally `cd $workingDirectory`)?

Reply:
Right, I will change the wording. I left it as a conditional to better detect and output a message if this does occur. It doesn't happen with `qsub` commands with HPC directives or interactive sessions.